apps/web/app/api/escrow/dispute/route.patch.ts

```
@@ -0,0 +1,45 @@
+                approved,
+                review_notes: review_notes ?? null,
+                reviewed_at: new Date().toISOString(),
+            })
+            .eq('id', id)
+            .select()
+            .single();
+        if (error) {
+            return NextResponse.json({ error: error.message }, { status: 500 });
+        }
+        return NextResponse.json({ dispute: data }, { status: 200 });
+    } catch (e: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)